python-tic-tac-toe-game-tkinter/README.md

```diff
@@ -0,0 +1,23 @@
+# Tic-Tac-Toe Game
+
+A Tic-Tac-Toe game built with Python and Tkinter. It allows two players to play a tic tac toe game by sharing the mouse.
```

Comment:
:writing_hand: **Writing Tip**

This should probably use hyphens per the [conversation](https://3.basecamp.com/4210154/buckets/11752590/chats/1729195627@5010541052) on Basecamp.

```suggestion
A Tic-Tac-Toe game built with Python and Tkinter. It allows two players to play a tic-tac-toe game by sharing the mouse.
```